integration/docker/run_test.go

```diff
@@ -112,13 +110,14 @@ func createLoopDevice() (string, string, error) {
 	}
 
 	// create loop device
-	for i := 0; i < losetupMaxTries; i++ {
-		loopPath := fmt.Sprintf("/dev/loop%d", i)
-		losetupCmd := NewCommand("losetup", "-P", loopPath, f.Name())
-		_, _, exitCode := losetupCmd.Run()
-		if exitCode == 0 {
-			return f.Name(), loopPath, nil
-		}
+	losetupCmd := NewCommand("losetup", "-fP", f.Name())
+	_, _, exitCode := losetupCmd.Run()
+	if exitCode == 0 {
+		getLoopPath := NewCommand("losetup", "-j", f.Name())
+		stdout, _, _ := getLoopPath.Run()
```

Comment:
I wonder if you should check the exit code

Reply:
fixed